mailutils.php

```diff
@@ -82,3 +83,65 @@ function mailutils_civicrm_navigationMenu(&$menu) {
   ));
   _mailutils_civix_navigationMenu($menu);
 }
+
+/**
+ * Implements hook_coreResourceList
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_coreResourceList
+ */
+function mailutils_civicrm_coreResourceList(&$list, $region) {
+  CRM_Core_Resources::singleton()->addStyleFile('mailutils', 'css/mailutils.css', 0, 'page-header');
+}
+
+/**
+ * Implements hook_civicrm_buildForm().
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_buildForm
+ */
+function mailutils_civicrm_buildForm($form_name, &$form) {
+  switch ($form_name) {
+    case 'CRM_Activity_Form_ActivityView': {
+      $activity_template_data = $form->getTemplateVars('values');
```

Comment:
In 5.69, this was still called `$form->get_template_vars()`; I used the snippet below for testing:

```
function _mailutils_get_template_vars(CRM_Core_Form $form, $name) {
  if (method_exists($form, 'getTemplateVars')) {
    return $form->getTemplateVars($name);
  }
  return $form->get_template_vars($name);
}
```